x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/profile/ProfileCancellationIntegTests.java

```diff
@@ -61,9 +84,19 @@ protected boolean addMockHttpTransport() {
     protected Collection<Class<? extends Plugin>> nodePlugins() {
         final List<Class<? extends Plugin>> plugins = new ArrayList<>(super.nodePlugins());
         plugins.add(SearchBlockPlugin.class);
+        {
+            // replace the security plugin with the security plugin with the dummy authorization engine extension
+            plugins.remove(LocalStateSecurity.class);
+            plugins.add(LocalStateWithDummyAuthorizationEngineExtension.class);
+        }
```

Comment:
I need this in order to block the "check privileges" invocation. This adds and dummy authorization engine (which authz everything) in this IT suite.